<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>DlgMapDirectionsBase</class>
  <widget class="QWidget" name="DlgMapDirectionsBase">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>509</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Directions</string>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <widget class="QGroupBox" name="m_fraDirectionsLong">
          <property name="title">
            <string>Long Directions Commands</string>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="3" column="3">
              <widget class="QLineEdit" name="m_txtNorthWestLong"/>
            </item>
            <item row="1" column="3">
              <widget class="QLineEdit" name="m_txtSouthEastLong"/>
            </item>
            <item row="2" column="1">
              <widget class="QLineEdit" name="m_txtSouthLong"/>
            </item>
            <item row="2" column="3">
              <widget class="QLineEdit" name="m_txtSouthWestLong"/>
            </item>
            <item row="0" column="1">
              <widget class="QLineEdit" name="m_txtNorthLong"/>
            </item>
            <item row="3" column="1">
              <widget class="QLineEdit" name="m_txtWestLong"/>
            </item>
            <item row="0" column="3">
              <widget class="QLineEdit" name="m_txtNorthEastLong"/>
            </item>
            <item row="5" column="1">
              <widget class="QLineEdit" name="m_txtDownLong"/>
            </item>
            <item row="1" column="1">
              <widget class="QLineEdit" name="m_txtEastLong"/>
            </item>
            <item row="4" column="1">
              <widget class="QLineEdit" name="m_txtUpLong"/>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="m_lblNorthLong">
                <property name="text">
                  <string>North</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="m_lblEastLong">
                <property name="text">
                  <string>East</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="m_lblSouthLong">
                <property name="text">
                  <string>South</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <widget class="QLabel" name="m_lblWestLong">
                <property name="text">
                  <string>West</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="0">
              <widget class="QLabel" name="m_lblUpLong">
                <property name="text">
                  <string>Up</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="5" column="0">
              <widget class="QLabel" name="m_lblDownLong">
                <property name="text">
                  <string>Down</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="2">
              <widget class="QLabel" name="m_lblNorthEastLong">
                <property name="text">
                  <string>North East</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="2">
              <widget class="QLabel" name="m_lblSouthEastLong">
                <property name="text">
                  <string>South East</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="2">
              <widget class="QLabel" name="m_lblSouthWestLong">
                <property name="text">
                  <string>South West</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="2">
              <widget class="QLabel" name="m_lblNorthWestLong">
                <property name="text">
                  <string>North West</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="m_fraDirectionsShort">
          <property name="title">
            <string>Short Directions Commands</string>
          </property>
          <layout class="QGridLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item row="3" column="3">
              <widget class="QLineEdit" name="m_txtNorthWestShort"/>
            </item>
            <item row="1" column="3">
              <widget class="QLineEdit" name="m_txtSouthEastShort"/>
            </item>
            <item row="2" column="1">
              <widget class="QLineEdit" name="m_txtSouthShort"/>
            </item>
            <item row="2" column="3">
              <widget class="QLineEdit" name="m_txtSouthWestShort"/>
            </item>
            <item row="0" column="1">
              <widget class="QLineEdit" name="m_txtNorthShort"/>
            </item>
            <item row="3" column="1">
              <widget class="QLineEdit" name="m_txtWestShort"/>
            </item>
            <item row="0" column="3">
              <widget class="QLineEdit" name="m_txtNorthEastShort"/>
            </item>
            <item row="5" column="1">
              <widget class="QLineEdit" name="m_txtDownShort"/>
            </item>
            <item row="1" column="1">
              <widget class="QLineEdit" name="m_txtEastShort"/>
            </item>
            <item row="4" column="1">
              <widget class="QLineEdit" name="m_txtUpShort"/>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="m_lblNorthShort">
                <property name="text">
                  <string>North</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="m_lblEastShort">
                <property name="text">
                  <string>East</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="m_lblSouthShort">
                <property name="text">
                  <string>South</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <widget class="QLabel" name="m_lblWestShort">
                <property name="text">
                  <string>West</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="0">
              <widget class="QLabel" name="m_lblUpShort">
                <property name="text">
                  <string>Up</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="5" column="0">
              <widget class="QLabel" name="m_lblDownShort">
                <property name="text">
                  <string>Down</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="2">
              <widget class="QLabel" name="m_lblNorthEastShort">
                <property name="text">
                  <string>North East</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="2">
              <widget class="QLabel" name="m_lblSouthEastShort">
                <property name="text">
                  <string>South East</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="2">
              <widget class="QLabel" name="m_lblSouthWestShort">
                <property name="text">
                  <string>South West</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="2">
              <widget class="QLabel" name="m_lblNorthWestShort">
                <property name="text">
                  <string>North West</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="Spacer2">
          <property name="sizeHint">
            <size>
              <width>0</width>
              <height>41</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
